pkgs/development/python-modules/pkgconfig/default.nix

```diff
@@ -19,11 +19,9 @@ buildPythonPackage rec {
 
   checkInputs = [ nose ];
 
-  nativeBuildInputs = [ pkgconfig ];
+  propagatedNativeBuildInputs = [ pkgconfig ];
 
-  checkPhase = ''
-    nosetests
```

Comment:
Why did you remove tests?

Reply:
Turns out the pypi distribution strips the test files. Also the project moved to pytest. I'll fix/rebase, then @ you when done.

Reply:
I just removed the test overrides – there's an upstream issue to include the test script in the pypi dist so next version bump should see tests working hopefully. Building form source is an option, but it uses "prose" which I have no experience with.

Reply:
I think we need to add ``doCheck = false;`` or otherwise it will run pytest with zero tests executed and to make sure it is importable please add ``pythonImportsCheck = [ "pkgconfig" ]``